numpy/_core/src/umath/string_ufuncs.cpp

```diff
@@ -208,37 +125,15 @@ string_len_loop(PyArrayMethod_Context *context,
 
 
 template <ENCODING enc>
-static int
-string_isalpha_loop(PyArrayMethod_Context *context,
-        char *const data[], npy_intp const dimensions[],
-        npy_intp const strides[], NpyAuxData *NPY_UNUSED(auxdata))
-{
-    PyArray_Descr *descr = context->descriptors[0];
-    int elsize = descr->elsize;
-
-    char *in = data[0];
-    char *out = data[1];
-
-    npy_intp N = dimensions[0];
-
-    while (N--) {
-        Buffer<enc> buf(in, elsize);
-        *(npy_bool *)out = (npy_bool) buf.isalpha();
-
-        in += strides[0];
-        out += strides[1];
-    }
-
-    return 0;
-}
-
+using buffer_method = bool (Buffer<enc>::*)();
```

Comment:
This seems nicer than my `typedef`! If you want to change it there too, that would be more than fine!